### PR TITLE
French RandomX Translation

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione,

Submitting @4593429418533 French (fr) translation he posted in #138. Made him clear in a comment that he should submit his work differently. 

Issue #165 

>:)

Ready for review, @el00ruobuob could you please take a look at this?

### CODE REVIEW

**Comment by el00ruobuob** [comment]
I'll have a closer look soon. And hope @4593429418533 will join the conversation. At first sight it seems there are a lot of adjustments to make (at least, to me) to make it fluent and consistent with other translations.

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Début [Nvidia (CUDA) benchmarks](https://github.com/SChernykh/RandomX_CUDA) révèle des augmentations de hashrate entre 100% et 150%, et cela devrait continuellement s'améliorer. Les travaux relatifs aux GPU AMD (OpenCL) sont en cours. Parce que RandomX est difficilement mémorisable, on s'attend à ce que les botnets et les malwares diminuent à mesure que la consommation de la mémoire sera plus facilement remarquée par les administrateurs. Une diminution globale du nethash augmentera la proportion de la prime globale versée aux mineurs légitimes, qu'ils travaillent avec des CPU ou des GPU. 
+
+RandomX a deux modes avec des besoins de mémoire et des performances différents. Le mode rapide nécessite 2 Go de mémoire partagée, mais 4x-6x les performances du mode lumière qui ne nécessite que 256 Mo de RAM. Le mode rapide est destiné aux mineurs dédiés. Le mode Light est conçu pour permettre aux nœuds complets de valider des blocs sans avoir besoin de 2+Go de RAM, de sorte que les petits périphériques (comme les ordinateurs monocartes ARM, par exemple Rock64) peuvent toujours être utilisés comme des nœuds autonomes.
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: `RandomX a deux modes` => `RandomX dispose de deux modes`  
`des besoins de mémoire` => `des besoins en mémoire`  
`mais 4x-6x les performances du mode lumière` => `mais offre 4 à 6 fois les performance du mode allégé`  
`Le mode rapide est destiné aux mineurs dédiés` => `Le mode rapide est destiné aux stations d'extraction minière dédiées`  
`Le mode Light` => `Le mode allégé`  
`que les petits périphériques` => `que de petits périphériques`  
`comme les ordinateurs monocartes ARM, par exemple Rock64` => `tel que des ordinateurs monocartes ARM comme Rock64`  
`peuvent toujours` => `puissent toujours`

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+RandomX a deux modes avec des besoins de mémoire et des performances différents. Le mode rapide nécessite 2 Go de mémoire partagée, mais 4x-6x les performances du mode lumière qui ne nécessite que 256 Mo de RAM. Le mode rapide est destiné aux mineurs dédiés. Le mode Light est conçu pour permettre aux nœuds complets de valider des blocs sans avoir besoin de 2+Go de RAM, de sorte que les petits périphériques (comme les ordinateurs monocartes ARM, par exemple Rock64) peuvent toujours être utilisés comme des nœuds autonomes. 
+
+| CPU | OS | Threads | RAM | CryptoNight-R (v8) | RandomX Fast Mode | RandomX Light Mode |
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: `Threads` => `Processus`  
`Fast Mode` => `mode rapide`  
`Light Mode` => `mode allégé`

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+|--|--|--|--|--|--|--|
+| AMD Ryzen 7 1700 | Ubuntu 16.04 | 8 | 16 GB DDR4 | 650 H/s | 4100 H/s | 620 H/s |
+| Intel Core i7-8550U | Windows 10 | 4 | 16 GB DDR4 | 240 H/s | 1700 H/s | 350 H/s |
+| Intel Core i3-3220 | Ubuntu 16.04 | 4 | 4 GB DDR3 | 75 H/s | 510 H/s | 150 H/s |  
+
+| GPU | Clockspeed | CryptoNight-R (v8) | RandomX |
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: `Clockspeed` => `Cadence`

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+|--|--|--|--|
+| GTX 1660 Ti | 2070/13760 MHz | 626 H/s (98 W) | 660 H/s (103 W) |
+| GTX 1080 Ti | 1930/10010 MHz | 787 H/s (145 W) | 1136 H/s (190 W) |  
+
+## _La Collaboration RandomX_ 
+
+RandomX a été développé pour Monero par tevador, hyc, vielmetti, antanst et SChernykh. D'autres organisations sont déjà intéressées par son adoption. [Arweave](https://www.arweave.org/), un facilitateur de stockage sans serveur a fait don de l'audit Trail of Bits à la communauté Monero et va implémenter RandomX avant Monero pour leur propre utilisation. Arweave fournit une approche innovante basée sur les cryptomonnaies pour le stockage décentralisé à long terme des données. L'exploitation minière sur Arweave repose à la fois sur la preuve du travail et - comme nouveau concept unique à Arweave - sur la preuve d'accès. Les mineurs d'Arweave sont incités par la preuve d'accès à la duplication et ont un accès rapide aux données stockées dans le réseau.
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: `L'exploitation minière` => `L'extraction minière`  
`la preuve du travail` => `une preuve de travail`  
`et - comme nouveau concept unique à Arweave - sur la preuve d'accès` => `et, c'est un nouveau concept propre à Arweave, sur une preuve d'accès`  
`sont incités par la preuve d'accès à la duplication et ont un accès rapide` => `sont incités par cette preuve d'accès à dupliquer et à avoir un accès rapide`  
`dans le réseau` => `sur le réseau`

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+[Wownero](http://wownero.org/) lance également RandomX dans sa prochaine mise à jour v0.6 et l'appellera RandomWOW. Le code RandomX sera mis à jour une fois les audits terminés, de façon a être prêt pour le prochain "fork" du réseau Monero. Une autre différence est que RandomWOW aura un scratchpad plus petit pour le hachage, 1Mo au lieu de 2Mo, un plus petit nombre d'itérations d'exécution de VM, et une augmentation des exécutions de VM en chaîne par hachage pour augmenter le coût de compilation du programme pour les GPU.
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: `de façon a être prêt pour le prochain "fork" du réseau Monero` => `il y aura donc quelques divergence du code lors de la prochaine mise à jour du réseau Monero`  
`Une autre différence est que RandomWOW aura un scratchpad plus petit pour le hachage` => `Une autre différence réside dans le presse-papier de hachage plus petit de RandomWOW`  
`un plus petit nombre d'itérations d'exécution de VM` => `le plus petit nombre d'itérations d'exécution de VMs`  
`et une augmentation des exécutions de VM en chaîne par hachage` => `et l'augmentation du chaînage des exécutions de VMs`  
`pour augmenter` => `afin d'augmenter`

**Comment by el00ruobuob** [comment]
Maybe if @assylias is still hanging around, he could give it a quick read too?

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## _Qu'est-ce qui va changer ?_
+
+Les ASIC sont les premières victimes de RandomX, comme il est optimisé pour les CPU, les GPU n'auront pas la même proportion de hashrate. 
+Début [Nvidia (CUDA) benchmarks](https://github.com/SChernykh/RandomX_CUDA) révèle des augmentations de hashrate entre 100% et 150%, et cela devrait continuellement s'améliorer. Les travaux relatifs aux GPU AMD (OpenCL) sont en cours. Parce que RandomX est difficilement mémorisable, on s'attend à ce que les botnets et les malwares diminuent à mesure que la consommation de la mémoire sera plus facilement remarquée par les administrateurs. Une diminution globale du nethash augmentera la proportion de la prime globale versée aux mineurs légitimes, qu'ils travaillent avec des CPU ou des GPU.
```
[the remark this answers is not in the document]
Comment (assylias) [reply monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: ils se feront plus facilement remarqués -> remarquer

File: monero-outreach-docs/translations/fr/storyboard/randomX_fr.md (hunk 5 above)
Comment (assylias) [reply monero-outreach-docs/translations/fr/storyboard/randomX_fr.md]: quelques divergence -> divergences

**Comment by lh1008** [comment]
Pinging @4593429418533 >:)

**Comment by 4593429418533** [comment]
> 
> 
> Pinging @4593429418533 >:)

Yes ?

**Comment by lh1008** [comment]
@4593429418533 there are some comments. Please answer them so we can continue with the reviewing/merging process.

**Comment by erciccione** [comment]
@4593429418533 This PR Is stalling. Could you check the comments to your translation and answer to them? we cannot merge until you tell us something.

I already spoke in private to @lh1008 about this but i'm going to reiterate it, to let all contributors know: it's not a good practice to submit somebody else's work. Makes reviews and corrections much harder and longer, better spend some time to learn how to correctly use github (or git). I and lh1008 are happy to help who needs it.

**Comment by lh1008** [comment]
Hey @erciccione,

Spoke with @4593429418533. He's not able to finish the comments. Spoke with him if we were able to continue without him and he has not answered yet. 

Any decision you make right now is okay. I understand what we spoke and agree with you that contributors should take time and learn how to submit work if they want to volunteer in the workgroup. 

Wouldn't want to @el00ruobuob or @assylias to feel disappointed because of this. I apologize for any inconvenience and delay in this. If anyone has any suggestions please share them.

**Comment by erciccione** [comment]
My suggestion on how to move forward is: 

- Give @4593429418533 half of the reward for the translation. Contributing to Outreach (as to any other workgroup), means follow the rules and workflow needed to have work submitted and reviewed. @4593429418533 didn't follow the process correctly.
- Give the other half of the reward to the French speaking person who will recover this PR and implement the changes suggested.
- Both reviews should be rewarded as normal.
- Make clearer to contributors that to receive the reward, they should follow the process from the beginning to end (create work -> submit it for reviews -> have it reviewed -> implement reviews), otherwise they will receive a partial reward or none at all. 

@el00ruobuob @assylias would one of you be up to recover this PR and implement your changes?

**Comment by lh1008** [comment]
I agree @erciccione with your suggestions. Will wait for @el00ruobuob and @assylias to respond to see how we continue with the PR.

**Comment by el00ruobuob** [comment]
I can finish the work myself, no problem.  
I just need to see how to recover a PR... taking the files and creating a new PR myself, adding @4593429418533 as co-author?

**Comment by lh1008** [comment]
Closing this PR and leaving PR #169 for the RandomX from Issue #165. Thank you @el00ruobuob

**Pull request closed without merging**